eclass/cmake.eclass

```diff
@@ -768,11 +583,11 @@ _cmake_emake_src_make() {
 
 }
 
-# @FUNCTION: cmake-utils_src_make
+# @FUNCTION: cmake_src_make
```

Comment:
To be honest, I always found the name `src_make` weird. Maybe `cmake_build`?

Reply:
and then, likewise `_cmake_ninja_src_make`, `_cmake_emake_src_make` to `cmake_*_build`?

Reply:
I don't know. I don't think they need to be public API. In fact, I would personally just inline them in some little `case ... in`.